src/WorkOS.net/Services/Organizations/_interfaces/CreateOrganizationOptions.cs

```diff
@@ -1,5 +1,6 @@
 ﻿namespace WorkOS
 {
+    using System;
```

Comment:
What is this `System` used for?

Reply:
It was added in order to use [`ObsoleteAttribute`](https://learn.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=net-8.0) which comes from the `System` namespace.